module/VuFind/src/VuFind/View/Helper/Root/IdentifierLinker.php

```diff
@@ -146,7 +167,14 @@ protected function checkContext()
      */
     public function isActive()
     {
-        $doi = $this->recordDriver->tryMethod('getCleanDOI');
-        return !empty($doi) && $this->checkContext();
+        $ids = $this->getIdentifiers();
+        $hasId = false;
+        foreach ($ids as $id) {
+            if ($id !== null) {
```

Comment:
If you implement my suggestion for `getIdentifiers` you could use `!empty($ids)` here. Otherwise, we should also check that `$id` is not `false`.